critique.py

```diff
@@ -115,13 +126,106 @@ def find_NX_class_nodes(self, parent, nx_class = 'NXentry'):
             if isNeXusGroup(node, nx_class):
                 node_list.append(node)
         return node_list
+    
+### Each method starting with "test_" will contribute a column to the results table.
+### The tests are conducted in alphabetical order.
+### Note that each test can write attributes onto self for later tests to use.
```

Comment:
This is a nice technique.  Still (a matter of preference), it would helpful to have some descriptive text in the function title, such as `test_01_FileType()`.  This still sorts but also provides _me_ better understanding of the code and workflow.